openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -25,6 +25,12 @@ class CourseOverview(django.db.models.Model):
     a course as part of a user dashboard or enrollment API.
     """
 
+    # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
+    VERSION = 1
```

Comment:
@adampalay The only solution I could think of would be computing a hash of CourseOverview.\__meta\__.fields and using that as the version identifier, and then having a test like:
```
def test_version_updated():
   asserEqual(CourseOverview.VERSION_HASH, CourseOverview.compute_version_hash())
```